app/assets/stylesheets/forever_style_guide/modules/_footer.scss

```diff
@@ -1,53 +1,118 @@
-.footer-header {
-  font-weight: 500;
+.footer-tagline {
+  padding-top: $padding-small-vertical;
+  padding-bottom: $padding-small-vertical;
 }
+
 .footer-main_block {
-  margin-top: 10px;
+  @media screen and (min-width: $screen-sm) {
+    padding-left: 0;
+    padding-right: 0;
+  }
+}
+
+.footer-main_block-nav_group-title {
+  text-transform: uppercase;
```

Comment:
You can use a a bootstrap utility for this.
https://getbootstrap.com/docs/3.3/css/#type-transformation